### ISSUE TITLE
Cannot serialise default date value

### ISSUE THREAD

**Issue by kevinhowbrook** [wagtail/wagtail-react-streamfield#40]
`errorObject of type builtin_function_or_method is not JSON serializable` is thrown when defining a block with a default date value e.g:

```
    start_time = blocks.DateTimeBlock(required=False, default=datetime.now)
```

**Comment by BertrandBordage** [comment]
Fixed in Wagtail by https://github.com/wagtail/wagtail/pull/4942/commits/0cb560980402342e52cdb8d9457e6d63a39e988b.
I will backport this change to wagtail-react-streamfield soon.

**Comment by kevinhowbrook** [comment]
Thanks @BertrandBordage

**Comment by BertrandBordage** [comment]
Fixed by c040db0395a0356b51d2303ddb8b376f1ac14877.
